eclass/freebsd.eclass

```diff
@@ -141,7 +147,8 @@ freebsd_src_unpack() {
 
 	# Starting from FreeBSD 9.2, its install command supports the -l option and
 	# they now use it. Emulate it if we are on a system that does not have it.
-	if [[ ${RV} > 9.1 ]] && ! has_version '>=sys-freebsd/freebsd-ubin-9.2_beta1' ; then
+	version_compare ${RV} 9.1
```

Comment:
Please avoid `version_compare` since it's hard to read. Prefer `! version_is_at_least`.